CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing To Cppcheck
+
+These are some guidelines *any* contributor should follow. They will help to make for better contributions which will most likely allow for them to be processed.
+
+(TODO: move daca@home section here?)
+
+## Code Changes
+
+Code contributions are handled via GitHub pull requests: https://github.com/danmar/cppcheck/pulls.
+
+If you file a pull request you might not get a reply immediately. We are a very small team and it might not fit in the current scope or time.
+
+Any kind of contribution is welcome but we might reject it. In that case we usually provide an explanation for the reasons but everything is always open to discussion.
+
+Changes in the `externals` folder need to be submitted to the respective upstream projects and will be pulled downstream with their next stable release (`picojson` is an exception because it is no longer maintained - the handling of such changes is not yet determined - see also https://trac.cppcheck.net/ticket/12233).
+
+Also after you filed a pull request please be ready to reply to questions and feedback. If you just "dump and leave" it might lower the chances of your change being accepted. This also applies after it was successfully merged as it might cause issues which were not exposed by the CI.
+
+Please be not discouraged if your change was rejected or if the review process might not have been as smooth as it could have been.
+
+Each change should be accompanied with a unit (C++) or integration (Python) test to ensure that it doesn't regress with future changes. Negative tests (testing the opposite behavior) would be favorable but might not be required or might already exist depending on the change. Tests which introduce `TODO_ASSERT_` or `@pytest.mark.skip`/`@pytest.mark.xfail` should have tickets filed.
```

Comment:
Maybe link to [C++](https://github.com/danmar/cppcheck/tree/main/test) and [Python](https://github.com/danmar/cppcheck/tree/main/test)?

Reply:
Done